src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let PRICE = days * 40;
+
+  if (days >= 7) {
+    PRICE -= 50;
+  } else if (days >= 3) {
+    PRICE -= 20;
```

Comment:
The current logic applies only one discount based on the rental period. According to the task requirements, if the rental period is 7 days or more, a $50 discount should be applied. If the rental period is 3 days or more but less than 7 days, a $20 discount should be applied. Ensure that the correct discount is applied based on these conditions.